src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -1429,11 +1433,11 @@ internal override void ForceComplete(SourceLocation locationOpt, CancellationTok
         }
     }
 
-    internal sealed class SourceComplexParameterSymbolWithCustomModifiersPrecedingByRef : SourceComplexParameterSymbol
+    internal sealed class SourceComplexParameterSymbolWithCustomModifiers : SourceComplexParameterSymbol
```

Comment:
what other subclasses of `SourceComplexParameterSymbol` are there? (I'll determine this as I continue reviewing, but feel free to answer this if you have a moment 😄)

Reply:
The other subclasses are `LambdaParameterSymbol` and `SynthesizedAccessorValueParameterSymbol`.